src/zendoo/mcTestCall.cpp

```diff
@@ -448,6 +452,9 @@ void create_verify(int argc, char** argv)
     if (circ_type_raw == "cert") {
         assert(argc >= 17);
         create_verify_test_cert_proof(ps_type_raw, argc, argv);
+    } else if (circ_type_raw == "cert_no_const") {
+        assert(argc >= 16);
+        create_verify_test_cert_proof(ps_type_raw, argc, argv);
     } else if (circ_type_raw == "csw") {
         create_verify_test_csw_proof(ps_type_raw, argc, argv);
```

Comment:
Let's take the occasion also here to assert correct size of params for csw, just for completeness. It should be:

`assert(argc >= 13 && argc <= 16);`

Reply:
Done.